domains/ahad.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "AhadWeb3",
+        "email": "ahadweb3.0@gmail.com"
+      },
+
+      "record": {
+        "URL": "ahad"
```

Comment:
Invalid URL

Reply:
https://ahadweb3.github.io/Ahad/

Hey!
will this work?